src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/resources/doclint.properties

```diff
@@ -77,8 +77,8 @@ dc.tag.nested.tag=nested tag: {0}
 dc.tag.not.allowed.here = tag not allowed here: <{0}>
 dc.tag.not.allowed = element not allowed in documentation comments: <{0}>
 dc.tag.not.allowed.inline.element = block element not allowed within inline element <{1}>: {0}
-dc.tag.not.allowed.inline.tag = block element not allowed within @{1}: {0}
-dc.tag.not.allowed.inline.other = block element not allowed here: {0}
+dc.tag.not.allowed.element.default.style = block element not allowed within element <{1}> with default style: {0}
```

Comment:
Is it possible to improve the wording?

As written, it is ambiguous/confusing: it seems like `{0}` is the default style, which I don't think is what you intend.
Also, inconsistent pointy brackets `<{1}>` but just `{0}`

Reply:
That looks good

Optionally, we could file a followup RFE to try and make more resources more stylistically consistent

Reply:
I changed the messages to the new format.